sample_workloads/nccltest/gke/values.yaml

```diff
@@ -0,0 +1,60 @@
+cluster:
+  nNodes: null  # Must specify on commandline (--set cluster.nNodes=2)
+  sbPlacement: false # Set `true` if running across multiple superblocks
+  nSuperblocks: 1
+  startSuperblock: 1
+  gcsBucket: "pirillo-sct-bucket"
+
+ncclBenchmarks:
+  image: "us-central1-docker.pkg.dev/supercomputer-testing/pirillo-gcr/nccltest"
+  tag: "latest"
+  # A comma-separated list of benchmarks to run.
+  benchmarks: "all_gather_perf,all_reduce_perf,sendrecv_perf"
+  # A comma-separated list of hex masks for the benchmarks.
+  # Must be of the same length as ncclBenchmarks.benchmarks.
+  # Each mask must be less than ncclBenchmarks.gpusPerNode.
+  masks: "0x0,0x0,0x0"
```

Comment:
For send/recv, make the mask 0x7 to make for a more representative traffic pattern.